### ISSUE TITLE
[BUG] Unexpected error message for invalid file type (based on accept)

### ISSUE THREAD

**Issue by pzi** [react-dropzone/react-dropzone#1221]
**Describe the bug**
When defining the accept param as per Docs/TypeScript definition:

```ts
```

**To Reproduce**

3. Select file '....'

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS]

**Smartphone (please complete the following information):**

**Additional context**

**Comment by PeterSHL** [comment]
I agree it would be really nice if it could revert to showing the file extensions as it did before.

The new error message is kinda cryptic :( 

`File type must be image/png,.png,image/gif,.gif,image/jpg,.jpg,.jpeg,image/bmp,.bmp,image/webp,.webp,text/csv,.csv,application/pdf,.pdf,application/msword,.doc,application/vnd.ms-excel,.xls,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,.xlsx,application/vnd.openxmlformats-officedocument.wordprocessingml.document,.docx.`

**Comment by rolandjitsu** [comment]
@pzi you're welcome to both provide a fix and enhancement for customising the messages.

**Comment by pzi** [comment]
Thanks for addressing this @rolandjitsu :) as mentioned in the other issue, we're still rolling with `v^11.7.1` as it satisfied what we needed. But, would be nice to upgrade eventually :)
